src/register.ts

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios'
```

Comment:
We may already be using 2 different http libraries. I think the chasqui stuff in uport-transports uses one and uport-lite uses another. It may not be an issue. But I like to keep the dependency count down. Have a look what the others use and/or investigate how large axios is. Use your best judgement.

Reply:
will take a look